src/DatePicker/DatePicker.js

```diff
@@ -150,11 +151,6 @@ class DatePicker extends React.Component {
     value: React.PropTypes.any,
 
     /**
-     * Creates a ValueLink with the value of date picker.
-     */
-    valueLink: React.PropTypes.object,
-
-    /**
      * Wordings used inside the button of the dialog.
      */
     wordings: deprecated(React.PropTypes.object, 'Instead, use `cancelLabel` and `okLabel`.'),
```

Comment:
in which case, if this DatePicker update goes in 0.15.0, `wordings` needs to stay. It was deprecated post 0.14.4, so is due for removal in 0.16.0-alpha.1.